presto-orc/src/main/java/com/facebook/presto/orc/OrcWriteValidation.java

```diff
@@ -282,19 +412,29 @@ public void validateRowGroupStatistics(
             throw new OrcCorruptionException(orcDataSourceId, "Unexpected row group %s in stripe at offset %s", rowGroupIndex, stripeOffset);
         }
 
+        // exclude stats for flat map keys because they are not present in the row group stats
+        ImmutableList.Builder<ColumnStatistics> actualAdjusted = ImmutableList.builder();
```

Comment:
This method gets actual stats from the blocks. We need to remove key node stats because they are not present in the ROW_INDEX.